### PR TITLE
Add alert when deeplink is invalid

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
When a deeplink is invalid, show an alert

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-53902

#### Release Note
```release-note
Added an alert when a deeplink is invalid
```

### CODE REVIEW

**Review by larkox — APPROVED** [review]

File: app/utils/deep_link/index.ts [hunk 1]
```diff
@@ -115,7 +115,11 @@ export async function handleDeepLink(deepLinkUrl: string, intlShape?: IntlShape,
 }
 
 export function parseDeepLink(deepLinkUrl: string): DeepLinkWithData {
-    const url = removeProtocol(deepLinkUrl);
+    const url = removeProtocol(decodeURIComponent(deepLinkUrl));
```
Comment (esarafianou) [inline app/utils/deep_link/index.ts:118]: Don't we need to catch the exception where URI is malformed: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent#exceptions ?

**Review by esarafianou — APPROVED** [review]
One small comment, otherwise looks good.

File: app/utils/deep_link/index.ts (hunk 1 above)
Comment (enahum) [reply app/utils/deep_link/index.ts:118]: thank you for spotting this @esarafianou

**Review by lindy65 — APPROVED** [review]
Thanks @enahum - I see the error "The link you are trying to open is invalid" 👍

**Pull request merged**